### PR TITLE
Introduce Mesos Stream connection

### PR DESCRIPTION
**Pull request by nLight**
This PR adds a ready to use `MesosStream` observable one can subscribe to to get the data. 

Opened against `drozhkov/introduce-ioc-container` for your convenience.

The observable is a `ReplaySubject` that will keep all previous values[<sup>1</sup>](#limit) and replay them to every new subscriber ensuring there's no data loss. 

`toConstantValue` IoC container will ensure you're getting the very same instance of the observable. 

`refCount` means that `rxjs` will take care counting subscribers and once there's no subscribers left it will terminate the observable. Once someone subscribes again it will open a new connection.

## Testing
a bit of manual work need s to be done. `src/js/index.js` could be a good place to add this snippet.

```javascript
import container from "./container";
import {MesosStreamType} from "./core/MesosStream";

// Hey container, give me what you have for this type
const mesosStream = container.get(MesosStreamType);

const subscription = mesosStream.subscribe(
  (value) => console.log(value),
  (err) => console.log(err)
);
```

## Dependencies
✅  #2463

## Footnotes
<a name="limit"></a> 1. Limited to `Number.MAX_SAFE_INTEGER`

### CODE REVIEW

**Comment by nLight** [comment]
Dependency has been merged, good to go 🙇

**Comment by nLight** [comment]
Ping!

**Comment by nLight** [comment]
Ping!

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

File: src/js/core/MesosStream.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { stream } from "mesos-client";
+
+export const MesosStreamType = Symbol("MesosStreamType");
```
Comment (orlandohohmeier) [inline src/js/core/MesosStream.js]: 🎨  The const name rubs me the wrong way, but I'm ok to merge as is.

**Pull request merged**